### ISSUE TITLE
Bug: MTU set too high, request for optional manual setting and warning notification.

### ISSUE THREAD

**Issue by firefoxlover** [mullvad/mullvadvpn-app#3875]
## Issue report

Operating system: Kubuntu (Ubuntu) 22.04

App version: 2022.3-beta2

### Issue description

Since I moved to a different internet connection, a lot of apps can`t send internet traffic anymore. This will be due to block size or something, the solution is this command:

```
sudo ip link set dev wg-mullvad mtu 1280
sudo systemctl enable systemd-resolved
sudo ln -sf /run/systemd/resolve/stub-resolv.conf /etc/resolv.conf
sudo systemctl start systemd-resolved
sudo systemctl restart NetworkManager
```

### Solution
So as long as the MTU-level is automatically set too high, there at least has to be a manual switch, as I have to enter this command (through an appstarter for more comfort) every few hours as it resets itself too high.

**Comment by pinkisemils** [comment]
This is a known bug and should be fixed in 2022.4. Are you experiencing the same issue there too?

The MTU set in the tunnel is inferred from the MTU set on the network interface used to send tunnel traffic - is the MTU on that interface not too high either?

**Comment by faern** [comment]
Closing this as it should be fixed in the latest stable release, as pointed out above. And no response from OP. Please re-open if this is still an issue.
